src/main/scala/rocket/CryptoSM.scala

```diff
@@ -0,0 +1,62 @@
+// See LICENSE.SiFive for license details.
```

Comment:
Actually this file is under the Apache license(not SiFive), although SiFive is Apache as well. 
```suggestion
// SPDX-License-Identifier: Apache-2.0
```